mscore/musescore.cpp

```diff
@@ -541,8 +541,8 @@ MuseScore::MuseScore()
 #endif
       viewModeCombo->setAccessibleName(tr("View Mode"));
       viewModeCombo->setFixedHeight(preferences.iconHeight + 8);  // hack
-      viewModeCombo->addItem(tr("Page View"));
-      viewModeCombo->addItem(tr("Continuous View"));
+      viewModeCombo->addItem(tr("Page Mode"));
```

Comment:
What's up with this word "Mode"? We use it for the note entry/edit/ etc... I think it would just cause more problems if we use it for more features...